static/src/stylesheets/pasteup/forms/_styles.scss

```diff
@@ -1,65 +1,7 @@
-@import '_forms.config';
-
-/*doc
----
-title: Forms
-name: forms
-category: Common
----
-
-```html_example
-<form class="form">
-    <fieldset class="fieldset">
-        <div class="fieldset__heading">
-            <h2 class="form__heading">
-                Form Heading
-            </h2>
-            <div class="form__note">
-                Form Note. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugiat, ratione!
-            </div>
-        </div>
-        <div class="fieldset__fields">
-            <ul>
-                <li class="form-field">
-                    <label class="label" for="field-1">Label</label>
-                    <input class="text-input" name="field-1" />
-                </li>
-                <li class="form-field">
-                    <label class="label" for="field-2">Another label</label>
-                    <input class="text-input" name="field-2" />
-                </li>
-                <li class="form-field">
-                    <div class="form-field__note">
-                        <a href="#">Forgotten your password?</a>
-                    </div>
-                    <button type="submit" class="submit-input">Submit</button>
-                    <label class="check-label check-label--helper">
-                        <input type="checkbox" />
-                        Remember me
-                    </label>
-                </li>
-            </ul>
-        </div>
-    </fieldset>
-</form>
-```
-*/
-
-/**
```

Comment:
the IE8 fix? i htink in sass is fine

On Wed, 13 Apr 2016 at 16:49 Sam Morris <notifications@github.com> wrote:

> In static/src/stylesheets/pasteup/forms/_styles.scss
> <https://github.com/guardian/frontend/pull/12553#discussion_r59573387>:
>
> > -                        <a href="#">Forgotten your password?</a>
> > -                    </div>
> > -                    <button type="submit" class="submit-input">Submit</button>
> > -                    <label class="check-label check-label--helper">
> > -                        <input type="checkbox" />
> > -                        Remember me
> > -                    </label>
> > -                </li>
> > -            </ul>
> > -        </div>
> > -    </fieldset>
> > -</form>
> > -```
> > -*/
> > -
> > -/**
>
> probably shouldn't live in the sass partial though. README or example.html
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/12553/files/b94203c96d07cc2d4f40ff51e0da77ef331e1221#r59573387>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396